networkit/cpp/algebraic/algorithms/test/AlgebraicTriangleCountingGTest.cpp

```diff
@@ -27,7 +27,7 @@ TEST(AlgebraicTriangleCountingGTest, testToyGraphOne) {
     AlgebraicTriangleCounting<CSRMatrix> atc(graph);
     atc.run();
 
-    std::vector<count> nodeScores = atc.getScores(true);
+    std::vector<count> nodeScores = atc.getScores();
```

Comment:
Theoretically one could use a `const auto& nodeScores =`. But since the test instance is quite small that's not strictly necessary.